pkg/configmap/store.go

```diff
@@ -0,0 +1,43 @@
+package configmap
```

Comment:
Is this file from serving? I'm curious why we need a defaulting config store.

Reply:
This is new for eventing. There is a TODO on line 8 to move this up to knative/pkg. 

We need a defaulting store because we can't rely on the desired ConfigMap existing. This is different than serving, where they install the ConfigMap at the same time they install their controllers. Eventing does the same for our controllers. However, the Broker Ingress is running in whichever namespace the user wants a Broker. We don't ensure the ConfigMap is present in that namespace. Instead, we rely on an code supplied default. If the ConfigMap is present, great! Use it. If not, then use the code supplied default.